omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+    * - Supported
+      - supported/suboptimal
+      - Older version which is tested, confirmed to work correctly, but may not offer optimal performance/experience
+    * - Dropped
+      - unsupported/old
+      - Old version not longer tested and no longer officially supported; may or may not work (use at own risk)
+    * - Broken
+      - unsupported/broken
+      - Known to not work
+    * - Unsupported
+      - unsupported/misc
+      - Not supported for some reason other than the above
+
+Operating system support
+------------------------
+
+The following subsections detail the versions of each operating system
+which are supported by both its upstream developers (for security and
+general updates) and by OME for OMERO building and server and client
+deployment.
+
+Microsoft Windows
+^^^^^^^^^^^^^^^^^
+
+`General overview <http://support.microsoft.com/gp/lifeselectindex#W>`__
+
+.. tabularcolumns:: |l|l|l|l|l|l|J|
+
+.. list-table::
+    :header-rows: 2
+
+    * - 
+      - 
+      - Upstream
+      - support
+      - 
+      - 
+      - 
+    * - Version
+      - Release date
+      - (mainline)
+      - (extended)
+      - OME (build)
+      - OME (deploy)
+      - Details
+    * - XP
+      - from Dec 2001
+      - to Apr 2014
+      - to Apr 2014
+      - Dropped
+      - Dropped
+      - `Link <http://support.microsoft.com/lifecycle/?p1=12757>`__
+    * - Server 2003 R2
+      - from May 2003
+      - to Jul 2010
+      - to Jul 2015
+      - Dropped
+      - Dropped
+      - `Link <http://support.microsoft.com/lifecycle/?p1=10394>`__
+    * - Vista
+      - from Jan 2007
+      - to Apr 2012
+      - to Apr 2017
+      - Dropped
+      - Dropped
+      - `Link <http://support.microsoft.com/lifecycle/?p1=11737>`__
+    * - Win 7
+      - from Oct 2009
+      - to Jan 2015
+      - to Jan 2020
+      - Recommended
+      - Supported
+      - `Link <http://support.microsoft.com/lifecycle/?p1=14481>`__
+    * - Server 2008 R2
+      - from May 2008
+      - to Jan 2018
+      - to Jan 2023
+      - Recommended
+      - Supported
+      - `Link <http://support.microsoft.com/lifecycle/?p1=17383>`__
+    * - Win 8
+      - from Oct 2012
+      - to Jan 2018
+      - to Jan 2023
+      - Supported
+      - Upcoming
+      - `Link <http://support.microsoft.com/lifecycle/?p1=16799>`__
+    * - Server 2012
+      - from Oct 2012
+      - to Jan 2018
+      - to Jan 2023
+      - Supported
+      - Upcoming
+      - `Link <http://support.microsoft.com/lifecycle/?p1=16526>`__
+
+XP is now unsupported upstream, and Server 2003 will be unsupported
+within the timeframe of 5.1 support.  Vista is likely supported but
+usage is minimal and it's untested, hence marked as dropped.  7/Server
+2008 and 8/Server 2012 are both supported and used.  Server 2008
+builds in place; no 2012 builds or testing in place yet.  No regular
+Windows CI deployment and testing in place other than manually.  Some
+manual testing on Windows 7 VMs and machines.
```

Comment:
@mtbc It's intentionally not Recommended because this implies we have CI deployment or some other form of regular testing in place.  We didn't when I started the document, but we might now?  Until we've got it running under server2012, that's still marked Upcoming intentionally as well.  There's a card for that.

@kennethgillen What's the current state of w2008/w2012 CI stuff, do you know?  Do we have a daily deployment of the OMERO build on w2008?  Is there a w2012 CI node yet?  Thanks.